### PR TITLE
Chronos:  add distributed training example

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

File: pyzoo/zoo/chronos/examples/distributed/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Use Chronos forecasters in a distributed fashion
+LSTM, TCN and Seq2seq users can easily train their forecasters in a distributed fashion to handle extra large dataset and utilize a cluster. The functionality is powered by Project Orca.
```
Comment (TheaperDeng) [inline pyzoo/zoo/chronos/examples/distributed/README.md:2]: LSTM, TCN and Seq2seq users can easily train their forecasters in a distributed fashion to handle extra large dataset and speed up the process (especially the training) by utilizing a cluster or pseudo-distribution on a single node. The functionality is powered by Project Orca.

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

**Comment by TheaperDeng** [comment]
remember to add test scripts

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Pip-ExampleTests-py36/62/
http://10.112.231.51:18888/job/ZOO-PR-Validation/6485/

**Pull request merged**